### PR TITLE
Add License

### PR DESCRIPTION
**Pull request by awgreene**

### CODE REVIEW

**Comment by awgreene** [comment]
(Possibly) addresses https://github.com/operator-framework/api/issues/40

**Comment by estroz** [comment]
This should probably be added to all go files.

**Comment by Bowenislandsong** [comment]
> A license header should probably be added to all go files.

@awgreene I think you can use https://github.com/elastic/go-licenser to address this comment.

**Comment by awgreene** [comment]
> @awgreene I think you can use https://github.com/elastic/go-licenser to address this comment.

That seems to work, have we used this tool in other repos? @njhale @ecordell is there Licensor that I should use given the planned contribution to CNCF?

**Comment by Bowenislandsong** [comment]
are we dropping red hat from the license? 

Could we please drop the year? I was not a fan of removing/updating those license years.

File: .reuse/dep5 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: The Operator Framework API
+Upstream-Contact: The Operator Framework Authors <agreene@redhat.com>
+Source: https://github.com/operator-framework/api
+
+# Sample paragraph, commented out:
+#
+# Files: src/*
```
Comment (awgreene) [inline .reuse/dep5:8]: Looks like we could use this instead of the make file, thoughs?

**Comment by dvattaka** [comment]
Hi, Is the operator-framework/api/ is Licensed under Apache License 2.0? If that's the case, could you please add the LICENSE file?

**Comment by estroz** [comment]
@awgreene would be nice to get this in soon.

**Comment by ankitathomas** [comment]
Closing this out, https://github.com/operator-framework/api/pull/57 add the Apache 2.0 License

**Pull request closed without merging**